tests/integration/_tmux_session.py

```diff
@@ -161,10 +164,14 @@ def interaction(
         ignore_within_response=None,
         timeout=300,
     ):
-        """interact with the tmux session
+        # pylint: disable=too-many-branches
```

Comment:
Is it a conscious choice not to have coma-separated disables?

Reply:
yeah, I prefer one liners